.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,33 @@
+name: Versioning
+
+on:
+  pull_request:
+
+jobs:
+  bump-version:
+    name: Check version has been bumped
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout to BASE
+        uses: actions/checkout@v4
+        with:
+          ref: dev
```

Comment:
We should be able to get the base branch from github's workflow api, to avoid hard coding dev here